cli/src/commands/auth/commands/login.ts

```diff
@@ -0,0 +1,54 @@
+import { existsSync, mkdirSync, readFileSync } from 'node:fs';
+import { writeFile } from 'node:fs/promises';
+import { Command, program } from 'commander';
+import yaml from 'js-yaml';
+import open from 'open';
+import pc from 'picocolors';
+import { configDir, configFile } from '../../../core/config.js';
+import { BaseCommandOptions } from '../../../core/types/types.js';
+import { performDeviceAuth, startPollingForAccessToken } from '../utils.js';
+
+export default (opts: BaseCommandOptions) => {
+  const loginCommand = new Command('login');
+  loginCommand.description('Login a user.');
+
+  loginCommand.action(async () => {
+    if (existsSync(configFile)) {
+      const data = yaml.load(readFileSync(configFile, 'utf8'));
+      const loginData = JSON.parse(JSON.stringify(data));
+      if (loginData && loginData?.expiresAt && new Date(loginData.expiresAt) > new Date()) {
+        console.log(pc.green('You are already logged in.'));
+        process.exit(0);
+      }
+    }
+
+    const resp = await performDeviceAuth();
+    if (!resp.success) {
+      program.error('Could not perform authentication. Please try again');
```

Comment:
Can we show some debug information? or is it logged before?

Reply:
i dont think there is something we could show

Reply:
ok